NGitLab.Mock/Config/GitLabHelpers.cs

```diff
@@ -0,0 +1,1279 @@
+            });
+        }
+
+        /// <summary>
+        /// Add group permission in project
+        /// </summary>
+        /// <param name="project">Project.</param>
+        /// <param name="group">Group fullname (required)</param>
+        /// <param name="level">Access level.</param>
+        public static GitLabProject WithGroupPermission(this GitLabProject project, string group, AccessLevel level = AccessLevel.Developer)
+        {
+            return Configure(project, _ =>
+            {
+                var permission = new GitLabPermission
+                {
+                    Group = group ?? throw new ArgumentNullException(nameof(group)),
+                    Level = level,
+                };
+
+                project.Permissions.Add(permission);
+            });
+        }
+
+        /// <summary>
+        /// Add milestone in project
+        /// </summary>
+        /// <param name="project">Project.</param>
+        /// <param name="title">Title (required)</param>
+        /// <param name="configure">Configuration method</param>
+        public static GitLabProject WithMilestone(this GitLabProject project, string title, Action<GitLabMilestone> configure)
+        {
+            return Configure(project, _ =>
+            {
+                var milestone = new GitLabMilestone
+                {
+                    Title = title ?? throw new ArgumentNullException(nameof(title)),
+                };
+
+                project.Milestones.Add(milestone);
+                configure(milestone);
+            });
+        }
+
+        /// <summary>
+        /// Add milestone in project
+        /// </summary>
+        /// <param name="project">Project.</param>
+        /// <param name="title">Title (required)</param>
+        /// <param name="id">Explicit ID (config increment)</param>
+        /// <param name="description">Description.</param>
+        /// <param name="dueDate">Due date time.</param>
+        /// <param name="startDate">Start date time.</param>
+        /// <param name="createdAt">Creation date time.</param>
+        /// <param name="updatedAt">Update date time.</param>
+        /// <param name="closedAt">Close date time.</param>
+        /// <param name="configure">Configuration method</param>
+        public static GitLabProject WithMilestone(this GitLabProject project, string title, int id = default, string description = null, DateTime? dueDate = null, DateTime? startDate = null, DateTime? createdAt = null, DateTime? updatedAt = null, DateTime? closedAt = null, Action<GitLabMilestone> configure = null)
+        {
+            return WithMilestone(project, title, milestone =>
+            {
+                if (id != default)
+                    milestone.Id = id;
+
+                milestone.Description = description;
+                milestone.DueDate = dueDate;
+                milestone.StartDate = startDate;
+                milestone.CreatedAt = createdAt ?? DateTime.Now;
```

Comment:
Add dates should be Utc

```suggestion
                milestone.CreatedAt = createdAt ?? DateTime.UtcNow;
```